frontend/src/components/CreateEvent/CreateEvent.jsx

```diff
@@ -10,7 +11,30 @@ function CreateEvent ({onClose, onCreate, onEdit, initialView, isEdit}) {
     const [location, setLocation] = useState('');
     const [allDay, setAllDay] = useState(false);
     const [participants, setParticipants] = useState('');
-
+    const [duration, setDuration] = useState(1);
+    const [recommendations, setRecommendations] = useState([]);
+
+    const fetchRecommendations = async () => {
```

Comment:
you should move the fetching portion of this function to utils, so it's not recreated on every render